app/models/solis_cloud_installation.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: solis_cloud_installations
+#
+#  amr_data_feed_config_id :bigint(8)        not null
+#  api_id                  :text
+#  api_secret              :text
+#  created_at              :datetime         not null
+#  id                      :bigint(8)        not null, primary key
+#  school_id               :bigint(8)        not null
+#  station_list            :jsonb
+#  updated_at              :datetime         not null
+#
+# Indexes
+#
+#  index_solis_cloud_installations_on_amr_data_feed_config_id  (amr_data_feed_config_id)
+#  index_solis_cloud_installations_on_school_id                (school_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (amr_data_feed_config_id => amr_data_feed_configs.id) ON DELETE => cascade
+#  fk_rails_...  (school_id => schools.id) ON DELETE => cascade
+#
+class SolisCloudInstallation < ApplicationRecord
+  belongs_to :school, inverse_of: :solar_edge_installations
+  belongs_to :amr_data_feed_config
+
+  has_many :meters, dependent: :destroy
```

Comment:
This would delete any historical data, so not sure we want to use `:destroy` here. That's what we're doing with SolarEdge